draft-ietf-quic-transport.md

```diff
@@ -805,8 +787,8 @@ an unsupported version.
 
 # Frames and Frame Types {#frames}
 
-The payload of cleartext packets and the plaintext after decryption of protected
-payloads consists of a sequence of frames, as shown in {{packet-frames}}.
+The payload of Initial, Handshake and all protected packets consists of a
```

Comment:
Since Initial and Handshake packets are also semi-protected, we might be able to get the specific list out entirely.  The payload of all packet types, after removing packet protection, consists of..."